src/test/resources/ide/language/js/Constructs.js

```diff
@@ -0,0 +1,13 @@
+
+var some_<warning>constnt</warning> = "1";
+
+class Some<warning>Classs</warning> {
+
+    constructor(config) {
+        this.member_varibleleel = config;
+    }
+
+    some_<warning>metod</warning>() {
+        console.log(this.member_varibleleel);
+    }
+}
```

Comment:
Setup at IDEA settings "Add newline at end of file" please

Reply:
Done